### PR TITLE
E2E-test listening

### LINKED ISSUE #180: Create end-to-end tests related to audio listening
See https://github.com/aabounegm/cast/issues/176#issuecomment-1088375282 for a list of requirements, section **Audio listening** (4 requirements)

**Acceptance criteria**:
- the tests are written in Cypress
- each requirement has its own test (its own `it()`)

### PR DESCRIPTION
**Pull request by illright**
Closes #180

I couldn't for the life of me figure out how to test the scrubbing bar. Apparently, testing those is not a thing in Cypress either. Screw it

### CODE REVIEW

File: cypress/support/expect-playing-audio.ts [hunk 1]
```diff
@@ -10,4 +10,14 @@ const expectPlayingAudio = () => {
   });
 };
 
-export default expectPlayingAudio;
+export const expectNoPlayingAudio = () => {
```
Comment (aabounegm) [inline cypress/support/expect-playing-audio.ts:13]: Instead of duplicating the entire function and changing just one word, couldn't the common functionality be factored out to another function?

File: package.json [hunk 2]
```diff
@@ -78,6 +79,7 @@
     "eslint-plugin-jest-dom": "^4.0.1",
     "eslint-plugin-svelte3": "^3.4.1",
     "eslint-plugin-testing-library": "^5.2.0",
+    "html-to-text": "^8.2.0",
```
Comment (aabounegm) [inline package.json:82]: Don't we already have `dompurify`? How is this different?

File: src/lib/widgets/bottom-bar/ui/bottom-bar.svelte [hunk 3]
```diff
@@ -29,7 +29,7 @@
 >
   <BottomBarNavItem name="Gallery" icon={IconVideoLibrary} href="/" showLabel={!hasAudio} />
   {#if hasAudio}
-    <MiniPlayer on:click={() => dispatch('miniplayer-click')} />
+    <MiniPlayer on:expand={() => dispatch('miniplayer-click')} />
```
Comment (aabounegm) [inline src/lib/widgets/bottom-bar/ui/bottom-bar.svelte:32]: Why did you rename one event but not the other? (referring to miniplayer-**click** event)

File: cypress/support/expect-playing-audio.ts (hunk 1 above)
Comment (illright) [reply cypress/support/expect-playing-audio.ts:13]: I tried playing with it but it just didn't work well in any way, so I decided to just let it go. Suggestions welcome

File: package.json (hunk 2 above)
Comment (illright) [reply package.json:82]: DOMPurify only strips dangerous tags, afaik

File: src/lib/widgets/bottom-bar/ui/bottom-bar.svelte (hunk 3 above)
Comment (illright) [reply src/lib/widgets/bottom-bar/ui/bottom-bar.svelte:32]: Couldn't be bothered refactoring more than what was necessary. Also to keep the diff clean. I don't think this is a problem
